public/index.html

```diff
@@ -7,7 +7,23 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <title>Gnosis Safe Multisig</title>
   </head>
+  <style>
+    .logo {
```

Comment:
I'd change the class name to something more unique to prevent any possible conflicts